packages/library/index.js

```diff
@@ -131,6 +131,12 @@ module.exports = (neutrino, opts = {}) => {
           .end()
         .end()
       .end()
+    .merge({
```

Comment:
I think these are still supposed to be wrapped in a debug:

```js
.when(neutrino.options.debug, (config) => {
  config.merge({
    stats: {
      maxModules: Infinity,
      optimizationBailout: true
    }
  });
})
```

Reply:
My bad...